cohorts/cohort.py

```diff
@@ -1336,24 +1340,57 @@ def plot_survival(self,
             default_threshold = None
         else:
             default_threshold = "median"
-        results = plot_kmf(
-            df=df,
-            condition_col=plot_col,
-            xlabel=survival_units,
-            ylabel="Overall Survival (%)" if how == "os" else "Progression-Free Survival (%)",
-            censor_col="deceased" if how == "os" else "progressed_or_deceased",
-            survival_col=how,
-            threshold=threshold if threshold is not None else default_threshold,
-            ax=ax,
-            ci_show=ci_show,
-            with_condition_color=with_condition_color,
-            no_condition_color=no_condition_color,
-            with_condition_label=with_condition_label,
-            no_condition_label=no_condition_label,
-            color_palette=color_palette,
-            label_map=label_map,
-            color_map=color_map,
-        )
+        if strata is None:
```

Comment:
i agree - at the time just needed to get this to work. i had an alternate approach in mind, will give it a try.